apiml-security-common/src/main/java/org/zowe/apiml/security/common/config/WebClientConfig.java

```diff
@@ -83,13 +88,15 @@ HttpClient getHttpClient(HttpClient httpClient, boolean useClientCert) {
     WebClient webClient(HttpClient httpClient) {
         return WebClient.builder()
             .clientConnector(new ReactorClientHttpConnector(getHttpClient(httpClient, false)))
+            .codecs(configurer -> configurer.defaultCodecs().maxInMemorySize(2 * 1024 * 1024))
```

Comment:
should be configurable?

Reply:
Probably yes, but for now it's needed, I want to validate what was the default in the v2 (or before the API Catalog refactoring) first.